tests/test_vmtkScripts/conftest.py

```diff
@@ -0,0 +1,301 @@
+## Program: VMTK
+## Language:  Python
+## Date:      January 10, 2018
+## Version:   1.4
+
+##   Copyright (c) Richard Izzo, Luca Antiga, All rights reserved.
+##   See LICENCE file for details.
+
+##      This software is distributed WITHOUT ANY WARRANTY; without even
+##      the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+##      PURPOSE.  See the above copyright notices for more information.
+
+## Note: this code was contributed by
+##       Richard Izzo (Github @rlizzo)
+##       University at Buffalo
+
+import pytest
+import os
+import vmtk.vmtkimagereader as imagereader
+import vmtk.vmtksurfacereader as surfacereader
+import vmtk.vmtkimagetonumpy as imagetonumpy
+import vmtk.vmtksurfacetonumpy as surfacetonumpy
+import vmtk.vmtkimagewriter as imagewriter
+import vmtk.vmtkimagecompare as compare
+from hashlib import sha1
+import copy
+
+
+@pytest.fixture(scope='function')
+def input_datadir():
+    '''
+    returns a path to the vmtk/tests/testData directory
+    '''
+    datadir = '/Users/rick/projects/vmtk/vmtk-test-data/input'
```

Comment:
Until I get the CMake external project set up, the paths to the test-data is hardcoded to my system. If you want to run these test locally, please change these to reflect the appropriate values on your system.